modules/azure/api_connectors/office365_set/connection.json

```diff
@@ -0,0 +1,40 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "_generator": {
+            "name": "bicep",
+            "version": "0.10.61.36676",
+            "templateHash": "11296295536176546337"
+        }
+    },
+    "parameters": {
+        "location": {
```

Comment:
This parameter is never used

Reply:
You're right. I've also removed the location param from the other two connectors as the module does not expose it as a param anyway.

+ changed parameters casing based on Jeffrey's tip, so terragrunt doesn't have to recreate connections with every apply